polar_route/vessel_performance/vessel_factory.py

```diff
@@ -20,6 +23,9 @@ def get_vessel(cls, config):
         vessel_requirements = {"SDA": (SDA, ["max_speed", "unit", "beam", "hull_type", "force_limit", "max_ice_conc",
                                              "min_depth"]),
                                "Slocum": (SlocumGlider, ["max_speed", "unit", "max_ice_conc", "min_depth"]),
+                               "BoatyMcBoatFace": (BoatyMcBoatFace, ["max_speed", "unit", "max_ice_conc", "min_depth"]),
+                               "TwinOtter": (TwinOtter, ["max_speed", "unit"]),
+                               "Windracer": (Windracer, ["max_speed", "unit","max_ice_conc", "min_depth"]),
```

Comment:
Change "min_depth" to "max_elevation" as this about obstacle avoidance not water depth.